index.md

```diff
@@ -46,6 +46,18 @@ extendr_module! {
 }
 ```
 
+On R's side, users can access to the above Rust functions as following:
```

Comment:
I think this addition should be moved to below the sentence "The `#[extendr]` attribute causes the compiler..."

Also: "... users can access the above Rust functions as follows:"

Reply:
Thanks, agreed.